cmd/root.go

```diff
@@ -9,12 +9,14 @@ import (
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/formatters"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/version"
 	"github.com/spf13/cobra"
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "preflight <container-image>",
-	Short: "Preflight Red Hat certification prep tool.",
+	Use:     "preflight <container-image>",
```

Comment:
For OpenScap scan we need the root privilege. So this should be `sudo preflight <container-image>`

Reply:
This was originally `preflight <container-image>`, and this PR doesn't change anything other than spacing (auto-alignment done by the linter) changed due to the new `Version` line being longer.

That said, I'm not sure I agree that we should indicate users should user `sudo` here. Given the enabled checks are configurable, and the check utilizing openscap is only a single check, I think it makes more sense to leave this as is (and document elsewhere if cases where `sudo` should be used).

Reply:
I believe we should indicate running all checks and HasMinimalVulnerabilities requires the root privilege, otherwise the policy execution will fail. I could probably take care of it in the other PR that I opened.